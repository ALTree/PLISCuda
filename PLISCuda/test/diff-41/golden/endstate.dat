subvolumes=41
species=1
time=1.001805

0 
1 
1 
10 
22 
41 
78 
132 
258 
440 
692 
1216 
1809 
2627 
3575 
4621 
5864 
7175 
7970 
8835 
8980 
8931 
8024 
7087 
6000 
4718 
3579 
2566 
1854 
1229 
697 
420 
249 
160 
76 
32 
18 
9 
2 
1 
1 
