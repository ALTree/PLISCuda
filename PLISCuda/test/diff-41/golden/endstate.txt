subvolumes=41
species=1
time=1.001935

0 
1 
6 
13 
12 
33 
71 
131 
263 
448 
708 
1179 
1812 
2600 
3638 
4733 
5956 
7017 
8169 
8744 
9067 
8698 
8059 
7160 
6050 
4720 
3542 
2630 
1766 
1183 
660 
408 
228 
153 
78 
40 
11 
10 
2 
1 
0 
